src/rust/plugin-bootstrap/src/plugin_bootstrap_init.rs

```diff
@@ -14,7 +20,25 @@ static PLUGIN_CONFIG_PATH: &str = "/etc/systemd/system/plugin.service.d/override
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let (_env, _guard) = grapl_config::init_grapl_env!();
 
-    let mut bootstrap_client = PluginBootstrapClient::from_env().await?;
+    let endpoint = std::env::var("PLUGIN_BOOTSTRAP_CLIENT_ADDRESS")?;
+    let plugin_bootstrap_polling_interval_ms: u64 =
+        std::env::var("PLUGIN_BOOTSTRAP_POLLING_INTERVAL_MS")?.parse()?;
+
+    tracing::info!(
+        message = "waiting 5s for plugin-bootstrap to report healthy",
```

Comment:
This message depends on the value of `plugin_bootstrap_polling_interval_ms`, doesn't it?

Reply:
No, this is that non-parametrized timeout value also